src/std/os/kqueue.ss

```diff
@@ -167,19 +179,24 @@ package: std/os
   (define-const EVFILT_PROC)
   (define-const EVFILT_SIGNAL)
   (define-const EVFILT_TIMER)
-  (define-const EVFILT_DEVICE)
+  (cond-expand
+    ((not darwin)
+     (define-const EVFILT_DEVICE))
+    (else))
```

Comment:
you don't need the empty else clause in gerbil!

Reply:
I have to deal with two `cond-expand`s in this file. Gerbil's and Gambit's.

Contrary to gerbil's cond-expand Gambit's requires an else clause. Otherwise it raises an error with the message "Unfulfilled 'cond-expand'".

The `cond-expand` here is in a `begin-foreign` block. The content of a `begin-foreign` block is, as I found out during compilation, not processed by the expander of gerbil but directly handed over to Gambit.

Reply:
oh ok, didn't notice that